pkg/web/scene/scene.go

```diff
@@ -120,6 +120,15 @@ func (s Scene) AccountsList() *api.AccountsList {
 	return nil
 }
 
+func (s Scene) AccountDetail() *api.Account {
+	if data, ok := s[APIData]; ok {
+		if out, ok := data.(*api.Account); ok {
+			return out
+		}
+	}
+	return nil
+}
```

Comment:
There has to be some kind of factory function that takes a type and will do this … (not part of the review, just my musings).